router-tests/subscriptions/http_subscriptions_test.go

```
@@ -303,14 +310,278 @@ func TestHeartbeats(t *testing.T) {
+			ctx, cancel := context.WithTimeout(t.Context(), eventWaitTimeout)
+			defer cancel()
+
+			response := openCountEmpSSESubscription(
+				t,
+				ctx,
+				xEnv.RouterClient,
+				xEnv.GraphQLRequestURL(),
+				false,
+				eventIntervalMilliseconds,
+			)
+			defer response.Body.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)